rest/route/host_spawn.go

```diff
@@ -224,7 +224,7 @@ func CheckUnexpirableHostLimitExceeded(userId string, maxHosts int) error {
 		return errors.Wrapf(err, "error counting number of existing non-expiring hosts for '%s'", userId)
 	}
 	if count >= maxHosts {
-		return errors.Errorf("can only have %d expirable hosts", maxHosts)
+		return errors.Errorf("can have at most %d unexpirable host(s)", maxHosts)
```

Comment:
Remove the fake plural - just leave it "hosts"

Reply:
This is gramatically incorrect for "1 hosts" but done.

Reply:
no laws on Evergreen 🎉